14_clusters/simple_torch_cluster_script.py

```diff
@@ -33,10 +33,10 @@
 # +--+--+  +--+--+  +--+--+
 # |Rank 1| |Rank 2| |Rank 3|
 # +-----+  +-----+  +-----+
-#
```

Comment:
What's wrong with these?

Reply:
Oh this is pretty minor, but: I like to put blank lines in each spot where there would be whitespace in a rendered document. It's a bit closer to the look of the rendered doc and (at least to me) suggests that this document isn't purely intended as source code.

Minor enough that I wouldn't've done it if I weren't already making changes.